web/src/pages/changelog.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+This is a stub file. To generate the changelog, build VAST and move the
+generated `CHANGELOG.md` file in the build directory to
+`src/pages/changelog.md`.
```

Comment:
This should be in .gitignore then.

Reply:
That is not correct. The stub file has to be in the repo, otherwise local docs build will not work without copying the changelog.

Reply:
Yes, my point is actually different. I'm saying keep the stub file _and_ add it to .gitignore, so generating it (which overwrites the stub) doesn't cause the repo to get dirty.

Reply:
We agreed over lunch to do this only if it turns out to be necessary, i.e., not now as part of this PR.